xscen/catalog.py

```diff
@@ -935,8 +935,8 @@ def subset_file_coverage(
     files_to_keep = []
     for period in periods:
         period_interval = pd.Interval(
-            date_parser(period[0]),
-            date_parser(period[1], end_of_period=True),
+            date_parser(str(period[0])),
```

Comment:
`standardize_periods` is called earlier in that function, so I don't think this is needed?

Reply:
Ah woups indeed. Hum, why did I need that in my local tests then? I'll investigate.